src/ORM/DataObject.php

```diff
@@ -3538,6 +3538,17 @@ public function requireDefaultRecords()
         $this->extend('requireDefaultRecords', $dummy);
     }
 
+    /**
+     * Invoked after every database build is complete (including after table creation and
+     * default record population).
+     *
+     * See {@link DatabaseAdmin::doBuild()} for context.
+     */
+    public function onAfterBuild()
+    {
+        $this->extend('onAfterBuild', $dummy);
```

Comment:
What is the purpose of `$dummy` here?

Reply:
No purpose, other than to follow the pattern of other extension points in this class